dev/tools/dartdoc.dart

```diff
@@ -106,8 +116,18 @@ dependencies:
     args,
     workingDirectory: 'dev/docs',
   );
-  printStream(process.stdout);
-  printStream(process.stderr);
+  printStream(process.stdout, prefix: 'dartdoc:stdout: ',
+    filter: kVerbose ? const <Pattern>[] : <Pattern>[
+      new RegExp(r'^generating docs for library '), // unnecessary verbosity
+      new RegExp(r'^pars'), // unnecessary verbosity
```

Comment:
i'm tempted to just remove these two. it was only to make it a little easier while i was going through the output.